vars/skipStage.groovy

```diff
@@ -104,10 +104,8 @@ boolean skip_build_on_landing_branch(String target_branch) {
 boolean skip_scan_rpms(String distro, String target_branch) {
     return already_passed() ||
            target_branch == 'weekly-testing' ||
-           skip_stage_pragma('scan-rpms', 'true') ||
-           (distro == 'centos-7' &&
-            (!paramsValue('CI_SCAN_RPMS_el7_TEST', true)) ||
-            skip_stage_pragma('scan-centos-rpms')) ||
+           rpmTestVersion() != '' ||
+           skip_stage_pragma('scan-rpms', 'false') ||
```

Comment:
`false` is already the default value here.  We only supply a second argument to `skip_stage_pragma()` if we are looking for non-default behaviour, for example, `true` when trying to detect if a usually-skipped stage is being requested.

My concern here with supplying `false` here would be some kind of future change in the default value of that argument that is assuming that every caller that wants default behaviour is not supplying a value for the second argument.  We'd have this one outlier that operates differently with the new change and then a hunt to figure out why.